Dockerfile

```diff
@@ -58,6 +57,19 @@ RUN apt-get update -qq \
   && apt-get clean \
   && rm -rf /var/lib/apt/lists/*
 
+ARG BASE_RELEASE=bullseye
+RUN apt-get update -qq \
+  && apt-get install -y gnupg2 wget vim \
```

Comment:
Why are gnupg2, wget, and vim added?

Reply:
Copied code from a solution I used for getting the right pg version from the pg repo. Wget does seem unnecessary since it uses curl, and I'm not sure why vim would be needed either. I can try removing these and see if it still works.